pages/conference/young-inspirers.vue

```diff
@@ -62,7 +62,7 @@
                                 <p class="speechBox__time">
                                     {{ speech.speechtime }}
                                     <img
-                                        class="speechBox__langIcon"
+                                        class="speechBox__lang--icon"
```

Comment:
`icon` does not seem like a modifier. 
```suggestion
                                        class="speechBox__langIcon"
```

Reply:
the rest LGTM btw

Reply:
Thanks for the reminder, I've fixed that. I am also curious about if there is a way to express  a state that if one instance is along with an icon or not?

Reply:
could be `--show` or `--hide`?

Reply:
Got it. Thanks.